src/services/member/test/index.test.ts

```diff
@@ -6,17 +6,20 @@ import { FastifyInstance } from 'fastify';
 import { HttpMethod, ItemType, MAX_USERNAME_LENGTH } from '@graasp/sdk';
 
 import build, { clearDatabase } from '../../../../test/app';
+import { AppDataSource } from '../../../plugins/datasource';
 import { DEFAULT_MAX_STORAGE } from '../../../services/item/plugins/file/utils/constants';
 import { FILE_ITEM_TYPE } from '../../../utils/config';
 import { CannotModifyOtherMembers, MemberNotFound } from '../../../utils/errors';
 import { ItemTestUtils } from '../../item/test/fixtures/items';
-import { MemberRepository } from '../repository';
+import { Member } from '../entities/member';
 import { saveMember, saveMembers } from './fixtures/members';
 
 // mock datasource
 jest.mock('../../../plugins/datasource');
 const testUtils = new ItemTestUtils();
 
+const rawRepository = AppDataSource.getRepository(Member);
```

Comment:
I need it because I use `raw` functions of the repository, that does not exist in our MemberRepository (ex `findOneBy`). Should we provide the raw repository from the class? Or proxy the function through our class?